docs/CORTX-S3 Server Quick Start Guide.md

```diff
@@ -190,7 +190,7 @@ Before your test your build, ensure that you have installed and configured the f
     - To install easy_install, run the command: ` yum install python-setuptools python-setuptools-devel`
 2. Ensure you've installed pip.
     - To check if you have pip installed, run the command: ` pip --version`
-    - To install pip, run the command: ` easy_install pip`
```

Comment:
If I understand 'easy_install' is a valid python way of installing
Is any advantage to doing it through yum.
Is this change validated?

Reply:
@yanqingfu

Reply:
didn't work

[root@ssc-vm-g3-rhev4-1122 cortx-motr]# easy_install pip
Searching for pip
Reading https://pypi.python.org/simple/pip/
Couldn't find index page for 'pip' (maybe misspelled?)
Scanning index of all packages (this may take a while)
Reading https://pypi.python.org/simple/
No local packages or download links found for pip
error: Could not find suitable distribution for Requirement.parse('pip')